mlflow/pipelines/steps/predict.py

```diff
@@ -0,0 +1,153 @@
+import logging
+import os
+import time
+from typing import Dict, Any
+
+import mlflow
+from mlflow.exceptions import MlflowException, BAD_REQUEST, INVALID_PARAMETER_VALUE
+from mlflow.pipelines.cards import BaseCard
+from mlflow.pipelines.step import BaseStep
+from mlflow.pipelines.utils.execution import get_step_output_path
+from mlflow.pipelines.utils.step import get_pandas_data_profile
+from mlflow.utils._spark_utils import _get_active_spark_session
+
+_logger = logging.getLogger(__name__)
+
+
+# This should maybe imported from the preprocessing step for consistency
+_INPUT_FILE_NAME = "dataset.parquet"
```

Comment:
should be `scoring.parquet` to be more specific?

Reply:
Will change.